libsignal-service/Cargo.toml

```
@@ -7,8 +7,8 @@ license = "GPLv3"
 readme = "../README.md"
 
 [dependencies]
-libsignal-protocol = { git = "https://github.com/Michael-F-Bryan/libsignal-protocol-rs" }
-zkgroup = { git = "https://github.com/signalapp/zkgroup" }
+libsignal-protocol = { git = "https://github.com/signalapp/libsignal-client" }
```

```suggestion
libsignal-client = { git = "https://github.com/signalapp/libsignal-client" }
```
It is confusing that this is a different name than the repo.

---

Not ours to choose. The repo contains several crates, among which `libsignal-protocol`.